hw1/02-palindrome.html

```diff
@@ -9,17 +9,18 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+  <body class="bg-primary">
+    <main class="container-fluid">
+      <section class="m-2 card d-flex bg-light shadow">
```

Comment:
With these bootstrap styles, the card stretches to fill the available width of the viewport: 
![image](https://github.com/PSUtblock/frontend-homework/assets/25037249/765d2ea7-88d6-4b79-a58a-9a0f4558e0ec)

You should consider using one of the bootstrap display classes or another method to limit the width of the element.